core/memory_tracker/cc/memory_tracker.h

```diff
@@ -16,16 +16,24 @@
 
 #include "core/cc/target.h"
 
-#if (TARGET_OS == GAPID_OS_LINUX) || (TARGET_OS == GAPID_OS_ANDROID)
-#define COHERENT_TRACKING_ENABLED
-
 #ifndef GAPII_MEMORY_TRACKER_H
 #define GAPII_MEMORY_TRACKER_H
 
-#include <signal.h>
-#include <stdlib.h>
-#include <sys/mman.h>
-#include <unistd.h>
+#define COHERENT_TRACKING_ENABLED 1
```

Comment:
nit: Indent nested pre-processor logic after `#`, it's really hard to follow without indents.